.github/workflows/e2e.yml

```diff
@@ -21,4 +21,4 @@ jobs:
           browser: chrome
           headless: true
         env:
-          CYPRESS_BASE_URL: https://wazimap-staging.netlify.app
\ No newline at end of file
+          CYPRESS_BASE_URL: https://staging--wazimap-production.netlify.app
```

Comment:
sorry, unrelated change - getting rid of the wazimap-staging app because I think Mila didn't realise you could do branch deploys